### PR TITLE
Add documentation for unified query builder, add merge charts

### PR DESCRIPTION
**Pull request by manototh**
For reviewer:

This PR changes many files, but it mainly rewrites https://axiom-mano-unified-query.mintlify.app/query-data/explore

### CODE REVIEW

**Comment by manototh** [comment]
@bdsqqq Could you please take a look when you have the time? Thanks in advance.

File: query-data/explore.mdx [hunk 1]
```diff
@@ -1,70 +1,209 @@
...
+
+Click <img src="/doc-assets/icons/options.svg" className="inline-icon" alt="View options icon" /> to access the following options for each chart:
+
+- In **Values**, specify how to treat null values.
+- In **Variant**, specify the chart type. Select from area, bar, or line charts.
+- In **Y-Axis**, specify the scale of the Y axis. Select from linear or log scales.
+
+#### Merge charts
+
+When you run a query that produces several charts, Axiom displays the charts separately. For example:
```
Comment (manototh) [inline query-data/explore.mdx]:
```suggestion
When you run a query that produces several visualizations, Axiom displays the charts separately. For example:
```

**Review by 54892 — APPROVED** [review]

**Review by bdsqqq — APPROVED** [review]
amazing work, want us to follow up on builder specific playground links for a next PR, but other than that this looks great.

question: are we making a conscious decision not to include images? the step by step nature of the docs along with the clear labelling helped me follow along but couldn't help but notice the lack of visuals

**Comment by manototh** [comment]
> amazing work, want us to follow up on builder specific playground links for a next PR, but other than that this looks great.
> 
> question: are we making a conscious decision not to include images? the step by step nature of the docs along with the clear labelling helped me follow along but couldn't help but notice the lack of visuals

@bdsqqq Thank you so much! I'm so happy you liked the general direction.

Builder-specific links to the Playground would be amazing. Is this technically possible now?

As for images, your point is a valid concern. I'm consciously trying to reduce the number of screenshots in our docs. The UI is evolving very fast and a high number of screenshots is just not maintainable in the long run. I tried to make up for this with many contentful links to the Playground. I consider this even better than screenshots. Images are static and costly to maintain, whereas the Playground is interactive and dynamically updated, and enables users to play with the product with ease.

File: query-data/explore.mdx [hunk 2]
```diff
@@ -1,70 +1,209 @@
...
+
+If the query includes group-by clauses, there is a row for each group. Hold the pointer over a group row to highlight the group’s data on time series charts. Select the checkboxes on the left to display data only for the selected rows.
+
+#### Configure chart options
+
+Click <img src="/doc-assets/icons/options.svg" className="inline-icon" alt="View options icon" /> to access the following options for each chart:
+
+- In **Values**, specify how to treat null values.
+- In **Variant**, specify the chart type. Select from area, bar, or line charts.
+- In **Y-Axis**, specify the scale of the Y axis. Select from linear or log scales.
```
Comment (manototh) [inline query-data/explore.mdx]:
```suggestion
- In **Y-Axis**, specify the scale of the vertical axis. Select from linear or log scales.
```

**Pull request merged**